### PR TITLE
Correct issue with draft course runs pointing to official runs

### PR DESCRIPTION
**Pull request by staubina**
[DISCO-1121]

[DISCO-1121]: https://openedx.atlassian.net/browse/DISCO-1121

### CODE REVIEW

**Review by Dillon-Dumesnil — APPROVED** [review]
LGTM. Glad that's all it was

File: course_discovery/apps/api/v1/tests/test_views/test_course_runs.py [hunk 1]
```diff
@@ -277,8 +275,26 @@ def test_create_fails_with_missing_fields(self):
         # Send nothing - expect complaints
         response = self.client.post(url, {}, format='json')
         self.assertEqual(response.status_code, 400)
+        print(response.data)
```
Comment (mikix) [inline course_discovery/apps/api/v1/tests/test_views/test_course_runs.py]: Whoops

**Review by mikix — APPROVED** [review]
Sure? Some of those test changes are because we are checking start and end in a different spot now, I gather?

File: course_discovery/apps/api/v1/tests/test_views/test_course_runs.py (hunk 1 above)
Comment (staubina) [reply course_discovery/apps/api/v1/tests/test_views/test_course_runs.py]: opps

**Comment by staubina** [comment]
> Some of those test changes are because we are checking start and end in a different spot now, I gather?

@mikix, when the serializer validates it checks the data again, but I use the course earlier than we check the serializer.

**Pull request merged**